src/test/java/org/broadinstitute/hellbender/engine/ReadsDataSourceUnitTest.java

```diff
@@ -573,7 +573,7 @@ public void testManuallySpecifiedIndices( final List<Path> bams, final List<Path
     public void testManuallySpecifiedIndicesWithCustomReaderFactory( final List<Path> bams, final List<Path> indices ) {
         final SamReaderFactory customFactory = SamReaderFactory.makeDefault().validationStringency(ValidationStringency.STRICT);
 
-        try ( final ReadsDataSource readsSource = new ReadsDataSource(bams, indices, customFactory) ) {
+        try ( final ReadsDataSource readsSource = new ReadsDataSource(bams, indices, customFactory, null, null) ) {
```

Comment:
It doesn't compile without this change. I'm guessing you mean you want me to add a 3-arg constructor?

The inputs here are not on the Google Cloud, so the wrapper should be ignored. To check this, my new test passes a wrapper that returns null (if the code tries to use that, it'll blow up). The test passes.